packages/clay-autocomplete/src/ClayAutocomplete.js

```diff
@@ -18,6 +18,8 @@ class ClayAutocomplete extends ClayComponent {
 	 * @return {Boolean} If the event has been prevented or not.
 	 */
 	_handleDataChange(event) {
+		this._handleFilteredItems(event);
```

Comment:
If we always keep the call to `updateData` when the query changes, we will have the use case when we do not want to make a request for each query change, in this case we will always be issuing the `dataChange` event without them being altered